docs/debuginfod.md

```diff
@@ -0,0 +1,37 @@
+# debuginfod client
```

Comment:
The default configured debuginfod server is not mentioned anywhere here. I think that's useful information.